src/main/java/com/mdice/apitaskmanager/controller/TaskController.java

```diff
@@ -38,6 +39,10 @@ public List<Task> findAllByTaskstatus(@PathVariable("status") TaskStatus status)
         return this.taskService.findAllByTaskStatus(status);
     }
 
+    @GetMapping("/getbyid/{id}")
+    public Optional<Task> getById(@PathVariable("id") long id){
```

Comment:
Esto tiene que devolver un Task a secas, no un Optional

```suggestion
    public Task getById(@PathVariable("id") long id){
```